kani-driver/src/cbmc_output_parser.rs

```diff
@@ -914,12 +987,23 @@ fn remove_check_ids_from_description(mut properties: Vec<Property>) -> Vec<Prope
     properties
 }
 
-/// Extracts the property class from the property string.
-///
-/// Property strings have the format `([<function>.]<property_class_id>.<counter>)`
-pub fn extract_property_class(property: &Property) -> Option<&str> {
-    let property_class: Vec<&str> = property.property.rsplitn(3, '.').collect();
-    if property_class.len() > 1 { Some(property_class[1]) } else { None }
+fn get_property_class(property: &Property) -> Option<String> {
```

Comment:
Can you please make these functions methods of Property? Also, can you remove the "get_" per Rust API guidelines: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Done.